pkg/app/app.go

```diff
@@ -543,7 +543,8 @@ func (a *App) PrintState(c StateConfigProvider) error {
 				return
 			}
 
-			fmt.Printf("---\n#  Source: %s\n\n%+v", run.state.FullFilePath(), stateYaml)
+			sourceFile, _ := run.state.FullFilePath()
```

Comment:
Not useful to handle error here, IMO

Reply:
We should try our best to deal with errors so as not to cause the loss of error information and make it difficult to troubleshoot.

Reply:
If error is returned here it changes the behaviour of the current flow, I consider it's as breaking change. Since the comment is not that critical I would not break the current flow. Since you insist I will return the error.